packages/apps/frontera/src/routes/src/components/CommandMenu/commands/organization/AddNewOrganizations.tsx

```diff
@@ -75,15 +72,6 @@ export const AddNewOrganization = observer(() => {
     store.ui.commandMenu.toggle('AddNewOrganization');
   };
 
-  useKeyBindings(
```

Comment:
why this change? User should be able to submit even if neither of inputs are focused

Reply:
by mistake i added back